teuthology/task/install.py

```diff
@@ -1179,6 +1116,28 @@ def task(ctx, config):
         ceph:
           sha1: ...
 
+
+    Debug packages may optionally be installed:
+
+    overrides:
+      install:
+        ceph:
+          debuginfo: true
+
+
+    Default package lists may be overridden:
```

Comment:
So if we override the list, we have to replace the contents wholesale?
So the full fix for the bug will include setting these overrides in ceph-qa-suite by copying the entire package list and then updating them?